rllib/utils/test_utils.py

```diff
@@ -605,48 +610,34 @@ def run_learning_tests_from_yaml(
                 if min_reward is not None:
                     e["stop"]["episode_reward_mean"] = min_reward
 
-            keys = []
-            # Generate the torch copy of the experiment.
-            if len(frameworks) == 2:
-                e_torch = copy.deepcopy(e)
-                e_torch["config"]["framework"] = "torch"
-                keys.append(re.sub("^(\\w+)-", "\\1-tf-", k))
-                keys.append(re.sub("-tf-", "-torch-", keys[0]))
-                experiments[keys[0]] = e
-                experiments[keys[1]] = e_torch
-            # tf-only.
-            elif frameworks[0] == "tf":
-                keys.append(re.sub("^(\\w+)-", "\\1-tf-", k))
-                experiments[keys[0]] = e
-            # torch-only.
-            else:
-                keys.append(re.sub("^(\\w+)-", "\\1-torch-", k))
-                experiments[keys[0]] = e
+            # Generate `checks` dict for all experiments (tf, tf2 and/or torch).
```

Comment:
Just a question: This means we'll include tf2 in our weekly learning tests as well as our nightly multi-gpu tests, correct? I think the multi-gpu one would fail since RLlib+tf2 does not support multi-gpu yet. Could you check this? We should probably have a way to specify, which frameworks to test in the yaml files.
Like:
```
...
  config:
    frameworks: [tf, torch]  # instead of "framework": expands `experiments_to_run`, then removes the "frameworks" key from the struct
```